tsdb/db_test.go

```diff
@@ -4704,6 +4605,38 @@ func TestOOODisabled(t *testing.T) {
 	require.Len(t, ms.oooMmappedChunks, 0)
 }
 
+func TestOOOScheduledCompact(t *testing.T) {
+	opts := DefaultOptions()
+  opts.OutOfOrderTimeWindow = 30 * time.Minute.Milliseconds()
```

Comment:
Indentation is missing here. Once you fix the conflict that github is complaining about, the CI will let you know where all it is missing :). You can just use go fmt to fix this.